packages/patternfly-4/react-core/src/components/Select/Select.js

```diff
@@ -219,12 +220,16 @@ class Select extends React.Component {
         </ChipGroup>
       );
     }
-
+    ouiaId = getOIUAUniqueId();
```

Comment:
the usage of a number increment is in any case always wrong as it does not identify the actual object